README.md

```diff
@@ -170,7 +170,11 @@ The authors of pyramid_openapi3 believe that the approach of validating a manual
 
 ## Running tests
 
-You need to have [pipenv](https://pipenv.readthedocs.io/) and Python 3.7, 3.8, or 3.9 installed on your machine. Then you can run:
+You need to have [poetry](https://python-poetry.org/) and Python 3.7 through 3.11 installed on your machine.
+
+Alternatively, if you use [nix](https://nix.dev/tutorials/declarative-and-reproducible-developer-environments), run `nix-shell` to drop into a shell that has everything prepared for development.
+
+Then you can run:
 
     $ make tests
```

Comment:
Easier to copy-pasta this, plus uses a lexer for syntax highlighting.

````suggestion
    ```shell
    make tests
    ```
````

Reply:
Thanks!